FireRender.Maya.Src/FireRenderRamp.h

```diff
@@ -25,8 +25,7 @@ namespace FireMaya
 		static void* creator();
 		static MStatus initialize();
 
-		frw::Value GetValue(const Scope& scope) const override; 
-		frw::ArithmeticNode ApplyUVType(const Scope& scope, frw::ArithmeticNode& source, int uvIntType) const;
```

Comment:
I see you moved ApplyUVType function declaration to RampNodeConverter.h

Please move its implementation to corresponding *.cpp file, because now its implementation is in FireRenderRamp.cpp file